doc/faq.md

```diff
@@ -215,6 +215,14 @@ For `.vue` single file components it uses the `vue-template-compiler`
 
 - which will be in your module dependencies if you're developing with Vue).
 
+### Q: Does this work with Svelte as well?
+
+**A**: Yes.
+
+For `.svelte` single file components it uses the `svelte2tsx`
```

Comment:
During the ci-cleanup I noticed it also needs the svelte compiler, so `svelte` would be required to be present in the same node_modules tree as dependency-cruiser as well.

But I guess that's kind of a given when you're working on a svelte project (check)?

Reply:
Yep, https://github.com/sverweij/dependency-cruiser/pull/393#discussion_r523383294